plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/navigator/DBNProject.java

```diff
@@ -175,6 +175,7 @@ protected DBNNode[] readChildNodes(DBRProgressMonitor monitor) throws DBExceptio
             project.ensureOpen();
         }
 
+        project.setProjectProperty(DBNProjectConstant.USER_DECLINE_DECRYPTION, false);
```

Comment:
You don't need to set it. It will be implicitly `false`.

Reply:
I need to set it to false in case the user would like to open the project manually after declining. That is why I need to set it

Reply:
Ah, so it resets it. I see. Okay.